cmake/packaging/Config.cmake.in

```diff
@@ -26,15 +26,16 @@ find_package(fastcdr REQUIRED)
 find_package(foonathan_memory REQUIRED)
 find_package(TinyXML2 QUIET)
 @FASTRTPS_PACKAGE_OPT_DEPS@
+@FASTRTPS_INSTALLER_DEPS_ANCILLARY@
 
 if(FASTDDS_STATIC)
-    # OpenSSL static libraries are not distributed with the installer
-    @FASTRTPS_PACKAGE_OPT_DEPS@
+    @FASTRTPS_INSTALLER_OPT_DEPS@
     include(${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-static-targets.cmake)
 else()
     include(${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-dynamic-targets.cmake OPTIONAL RESULT_VARIABLE DYNAMIC_TARGET_FILE)
```

Comment:
When linking against our dynamic libraries, why is not done a `find_package(OpenSSL)` as before?

Reply:
The OpenSSL dependency is useless for dlls both from sources and binaries. Template config changed accordingly.